code/game/gamemodes/cult/runes.dm

```diff
@@ -1063,10 +1096,24 @@
 			if(iscultist(C) && !C.stat)
 				C.say("N'ath reth sh'yro eth d[pick("'","`")]rekkathnor!")
 				C.take_overall_damage(15, 0)
+				if(C != cultist)
+					C << "<span class='warning'>Your body take its toll as you drag your fellow cultist through dimensions.</span>"
+				else
+					C << "<span class='warning'>You feel a sharp pain as your body gets dragged through dimensions.</span>"
 		user.visible_message("<span class='warning'>The rune disappears with a flash of red light, and in its place now a body lies.</span>", \
 		"<span class='warning'>You are blinded by the flash of red light! After you're able to see again, you see that now instead of the rune there's a body.</span>", \
 		"<span class='warning'>You hear a pop and smell ozone.</span>")
 		qdel(src)
+	else
+		var/text = "<span class='sinister'>The following individuals are living and conscious followers of the Geometer of Blood:</span>"
+		for(var/mob/living/L in player_list)
+			if(L.stat != DEAD)
```

Comment:
No I mean "conscious" as in "not braindead, not catatonic". I want sleeping cultists to show up as well.